<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>maintenance</class>
 <widget class="QMainWindow" name="maintenance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1051</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #003366;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="settingsPic">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>120</y>
        <width>281</width>
        <height>221</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>settings.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>60</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 35px;</string>
      </property>
      <property name="text">
       <string>Maintenance</string>
      </property>
     </widget>
     <widget class="QPushButton" name="menuSouvenirButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>380</y>
        <width>171</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:DarkGrey ;
color: white;
border-radius: 10px;
}
QPushButton:hover:!pressed
{
background: Gray;
color: white;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>Modify Souvenirs</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MenuTeamButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>380</y>
        <width>171</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:DarkGrey ;
color: white;
border-radius: 10px;
}
QPushButton:hover:!pressed
{
background: gray;
color: white;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>Modify Teams</string>
      </property>
     </widget>
     <widget class="QPushButton" name="menuDistanceButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>380</y>
        <width>171</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:DarkGrey ;
color: white;
border-radius: 10px;
}
QPushButton:hover:!pressed
{
background: Gray;
color: white;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>Modify Distances</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 25px;</string>
      </property>
      <property name="text">
       <string>Profile: Admin</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="teamBackButton">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>550</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:DarkGrey ;
color: white;
border-radius: 10px;
}
QPushButton:hover:!pressed
{
background: Gray;
color: white;
border-radius: 10px;
}color: white;</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addTeamButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>510</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:DarkGrey ;
color: white;
border-radius: 10px;
}
QPushButton:hover:!pressed
{
background: Gray;
color: white;
border-radius: 10px;
}color: white;</string>
      </property>
      <property name="text">
       <string>Add Team File</string>
      </property>
     </widget>
     <widget class="QTableView" name="teamMaintenanceTableView">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>11</y>
        <width>1031</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QScrollBar{
	border: 1px solid #999999;
	background: white;
	width: 10px;
	margin: 0px 0px 0px 0px;
}
QScrollBar::handle:vertical{
	background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop: 0 rgb(32, 47, 130), stop: 0.5 rgb(32, 47, 130), stop:1 rgb(32, 47, 130));
	min-height: 0px;
}
</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>421</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="title">
       <string>Add Souvenir</string>
      </property>
      <widget class="QLabel" name="SouvenirLabelName">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Product name:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Product Cost:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="SouvenirNameLineEdit">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>271</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: white;
border: 1px solid Gray;
border-radius: 5px;
color: #003366;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Product Team:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="SouvenirCostSpin">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>271</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QComboBox" name="SouvenirTeamCombo">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>100</y>
         <width>271</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: white;
border: 1px solid Gray;
border-radius: 5px;
color: #003366;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>130</y>
         <width>80</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background:DarkGrey ;
color: white;
border-radius: 10px;
}
QPushButton:hover:!pressed
{
background: Gray;
color: white;
border-radius: 10px;
}color: white;</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="souvenirBackButton">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>550</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:DarkGrey ;
color: white;
border-radius: 10px;
}
QPushButton:hover:!pressed
{
background: Gray;
color: white;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QTableView" name="SouvenirTableView">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>11</y>
        <width>1021</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>410</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Show:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="showTeamsCombo">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>401</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: white;
border: 1px solid Gray;
border-radius: 5px;
color: #003366;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="distanceBackButton">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>550</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:DarkGrey ;
color: white;
border-radius: 10px;
}
QPushButton:hover:!pressed
{
background: Gray;
color: white;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QTableView" name="distancesTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1031</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addDistancesButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>530</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:DarkGrey ;
color: white;
border-radius: 10px;
}
QPushButton:hover:!pressed
{
background: Gray;
color: white;
border-radius: 10px;
}color: white;</string>
      </property>
      <property name="text">
       <string>Add distances file</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
